doc/classes/VisualScriptConstant.xml

```diff
@@ -1,10 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisualScriptConstant" inherits="VisualScriptNode" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
-		A Visual Script node which returns a constant value.
+		Gets a contant's value.
 	</brief_description>
 	<description>
-		A Visual Script node which returns the specified constant value.
+		This node returns a constant's value.
```

Comment:
User defined constant? Where do you define it? Maybe its worth adding this info

Reply:
You define it in the inspector, since it is a property of the node.